sdc/crypto/jwe_helper.py

```diff
@@ -0,0 +1,61 @@
+from jwcrypto import jwe
+from jwcrypto.jwe import InvalidJWEData
+from structlog import get_logger
+
+from sdc.crypto.exceptions import InvalidTokenException
+from sdc.crypto.helper import extract_kid_from_header
+
+logger = get_logger()
+
+
+class JWEHelper:
+    '''
+    Helper methods for encrypting a JWS/JWT. This class assumes you have used the RSA-OAP-A256GCM algoritm
+    and the required private key is available in the keystore.
+    '''
+    @staticmethod
+    def decrypt(encrypted_token, key_store=None, purpose=None, key=None):
```

Comment:
Actually I had that originally but didn't like the decrypt_with_key method name. This is where I want function overloading.

I don't want to change the kwargs order as that's a api breaking change. So i'll add the new method as Andrew suggested.